changelogs/URBAN_ATH_DICTIONARY_CHANGELOG.md

```diff
@@ -2,6 +2,39 @@
 
 We define here in what versions of the data dictionaries contains which tables
 
+## 1.5
```

Comment:
I would suggest to create a urban_ath 2_0 release instead of 1_5, this because of the breaking change of replacing urb_deg_ with urb_deg_t